bubble-wrap-style.yaml

```diff
@@ -2446,37 +2436,154 @@ layers:
                         - shield_text: true
                 draw:
                     mapzen_icon_library:
-                        #texture: mapzen_icon_library_shields_usa
+                        # texture: mapzen_icon_library_shields_usa
                         # you need to match any custom shield to the vector tile `network` values
-                        sprite: function() { return (feature.network + '-' + feature.shield_text.length + 'char'); }
-                        # FYI: sprite_default doesn't support functions, default is carried by parent style's sprite function
+                        sprite_default: |
```

Comment:
Note `sprite_default` doesn't support functions, I think this logic was moved up to the top of the shield layer now?

Reply:
Yeah, this section doesn't look right.